Queue.php

```diff
@@ -56,7 +63,13 @@ protected function getChannel()
     {
         if (null === $this->channel) {
             $this->channel = $this->connection->channel();
-            $this->channel->queue_declare($this->queueName, false, false, false, false);
+
+            $this->declareQueue($this->queueName);
+
+            if ($this->revisitDisabled == false) {
+                $this->declareQueue('revisit');
```

Comment:
Its covered there https://github.com/simgroep/concurrent-spider-bundle/pull/98/files#diff-3a9e0d716c811519f409dd6ea6e321adR124 and there https://github.com/simgroep/concurrent-spider-bundle/pull/98/files#diff-3a9e0d716c811519f409dd6ea6e321adR86